bin/pygrb/pycbc_make_offline_grb_workflow

```diff
@@ -234,12 +236,17 @@ if wflow.cp.has_option("workflow-condition_strain", "do-gating"):
         gated_frames.convert_to_lal_cache().tofile(
                             open(gated_cache.storage_path, "w"))
         datafind_files.append(gated_cache)
-
 datafind_veto_files.extend(datafind_files)
-ifo_list = sorted(sciSegs.keys())
-ifo = ifo_list[0]
-ifos = ''.join(ifo_list)
-wflow.ifos = ifos
```

Comment:
I saw the code that uses this after posting the comment. This looks correct because `wflow` has a `ifos_string` attribute which turns the list into a string.